client/my-sites/domains/domain-management/edit/mapped-domain.jsx

```diff
@@ -100,12 +105,20 @@ const MappedDomain = createReactClass( {
 
 					{ this.getAutoRenewalOrExpirationDate() }
 
-					<SubscriptionSettings
-						type={ domain.type }
-						subscriptionId={ domain.subscriptionId }
-						siteSlug={ selectedSite.slug }
-						onClick={ this.handlePaymentSettingsClick }
-					/>
+					<div className="edit__domain-inline-actions">
+						<SubscriptionSettings
+							type={ domain.type }
+							subscriptionId={ domain.subscriptionId }
+							siteSlug={ selectedSite.slug }
+							onClick={ this.handlePaymentSettingsClick }
+						/>
+
+						<Button href={ getDomainTransferPath( selectedSite.slug, domain.name ) }>
+							{ this.props.translate( 'Request Transfer', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Request Transfer Code' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
👎